### DISCUSSION TITLE
Any casing ideas?

### DISCUSSION THREAD

**Discussion by jeff464** [Jaffe-/NESizer2#25] [Q&A]
I'm in the final stages of building the NESizer and was wondering if there were plans on any sort of casing. For home use, the exposed PCB works, but this might prove troublesome when playing out (especially the smaller buttons). 

If not, I'm willing to learn 3D software if it means a sturdy case design - just not sure where to begin measuring. :)

**Comment by Jaffe-** [comment]
A case sounds really cool, I remember looking into that and quickly deciding that I'd leave it to someone else :p So I at least have not looked more into that, but I don't know what others have done!
